git/plan.sh

```diff
@@ -1,22 +1,32 @@
 pkg_name=git
 pkg_version=2.7.4
 pkg_origin=core
-pkg_license=('gplv2')
+pkg_description="Git is a free and open source distributed version control
+  system designed to handle everything from small to very large projects with
+  speed and efficiency."
```

Comment:
I've been going to the upstream_url and grabbing whatever they have as the first paragraph. Multi-line is fine.